azuredevops/resource_variable_group.go

```
@@ -94,6 +96,12 @@ func resourceVariableGroupCreate(d *schema.ResourceData, m interface{}) error {
 	}
 
 	flattenVariableGroup(d, addedVariableGroup, projectID)
+
+	// Update Allow Access
+	definitionResourceReferenceArgs := expandDefinitionResourceAuth(d, addedVariableGroup)
+	definitionResourceReference, err := updateDefinitionResourceAuth(clients, definitionResourceReferenceArgs, projectID)
```

We should return the error here if this fails

---

fixed